.bazelignore

```diff
@@ -3,3 +3,4 @@
 .cache
 orc8r/cloud/docker/metrics-configs/alert_rules
 nms/app/node_modules
+nms/node_modules/jshint/bin
```

Comment:
We can just ignore all of node_modules, I think

Reply:
No pretty unfortunate, I looked into it 😞
https://github.com/bazelbuild/bazel/pull/13756